src/components/NormalRightRow/index.jsx

```diff
@@ -1,37 +1,70 @@
 import React from 'react';
 import { shape, string } from 'prop-types';
+import Typography from '@material-ui/core/Typography';
 
 function NormalRightRow({ schema, classes }) {
   const keywords = Object.keys(schema);
+  const nonDisplayedKeywords = ['items', 'contains'];
+
+  // TODO: specify details to see below?
+  /*
+  if (typeof schema.additionalItems === 'object') {
+  }
+  */
 
   return (
     <div className={`${classes.row} ${classes.rightRow}`}>
       <div className={classes.keywordColumn}>
-        {keywords.map(keyword => (
-          <p key={keyword} className={classes.line}>
-            {keyword}
-            {': '}
-            {schema[keyword]}
-          </p>
-        ))}
+        {keywords.map(
+          keyword =>
+            !nonDisplayedKeywords.includes(keyword) && (
+              <Typography
+                component="div"
+                key={keyword}
+                className={classes.line}>
```

Comment:
See other comment about changing the variant.

Reply:
Applied changes!